lib/main_common.pm

```diff
@@ -501,12 +501,12 @@ sub load_extra_tests() {
             # sysauth test scenarios run in the console
             loadtest "sysauth/sssd";
         }
-        # finished console test and back to desktop
-        loadtest "console/consoletest_finish";
-        # kdump is not supported on aarch64, see BSC#990418
+        # kdump is not supported on aarch64, see bsc#1037407
         if (!check_var('ARCH', 'aarch64')) {
```

Comment:
@foursixnine what do you mean "we can start testing it". [bsc#990418](https://bugzilla.suse.com/show_bug.cgi?id=990418) is "RESOLVED WONTFIX" and [bsc#1037407](https://bugzilla.suse.com/show_bug.cgi?id=1037407) is still open.

@yxususe in this PR please focus on one thing at a time. In this case I recommend to not enable testing on something that can not work. We can later on add it with a special switch to add a scenario to the test development group. We can hardly handle more "known" failures and rather should focus on having some more green jobs first.